rules/dates-do-you-keep-date-formats-consistent-across-your-application/rule.md

```diff
@@ -44,10 +44,23 @@ Figure: Bad Example - Using hard coded formatting string
 'Initial CultureInfo settings for the application
 Public initialCulture As CultureInfo
 ...
+
 ...
 txtDateCreate.Text = CType(txtDateCreate.Tag, System.DateTime).ToString(initialCulture.DateTimeFormat)
 ```
 
 ::: good
 Figure: Good Example - Using culture info to format datetime
 :::
+
+## Browser language settings
+
+It is required to set web browser language to English (Australia) after modifying the computer reginal and language settings. This ensures proper formatting and compatibility, particularly when downloading web pages or accessing specific web content. 
+
+For example, Power BI is date format agnostic and will honor the setting in your browser. If your browser language is set to US English, it may cause date formatting issues.
+
+![Figure: Date format incorrect due to unmodified web page language settings](https://github.com/user-attachments/assets/45a1c7a2-ab73-4053-a3c6-3a19f09895fc)
```

Comment:
Hi @ClaraFang-SSW 

I see you added 2 images ehre, but they are not in the PR

- [ ] 1. Please make sure you add them to the rule folder, and fix the path (ping me if in doubt)